docs/motivating_use_case.md

```diff
@@ -190,8 +190,8 @@ appropriate party; we only show the claim here.
 
 ```json
 {
-  "attestation_type": "https://example.com/GitHubActionProduct/v1",
-  "subject": { "container_image_digest": "sha256:c201c331d6142766c866..." },
+  "predicateType": "https://example.com/GitHubActionProduct/v1",
+  "subject": [{ "container_image_digest": "sha256:c201c331d6142766c866..." }],
   "relations": {
```

Comment:
Is this `relations` field an extension field at the `Statement` layer? Might want to add a comment to denote that.